README.md

```diff
@@ -72,13 +72,12 @@ To build `workerd`, you need:
 * [Bazel](https://bazel.build/)
   * If you use [Bazelisk](https://github.com/bazelbuild/bazelisk) (recommended), it will automatically download and use the right version of Bazel for building workerd.
 * On Linux:
-  * Clang 11+ (e.g. package `clang` on Debian Bullseye)
-  * libc++ 11+ (e.g. packages `libc++-dev` and `libc++abi-dev` on Debian Bullseye)
-  * LLD 11+ (e.g. package `lld` on Debian Bullseye)
+  * Clang 15+ (e.g. package `clang-15` on Debian Bookworm). Earlier versions of clang starting with 12 may still work, but are not officially supported.
+  * libc++ 15+ (e.g. packages `libc++-15-dev` and `libc++abi-15-dev`)
+  * LLD 11+ (15 or higher is recommended, e.g. package `lld-15`).
```

Comment:
Suggest being consistent on 15 here too. We're not looking to spend time on older versions. There's a comment above about may still work, that could move into the 'On Linux:' intro